pages/groups/group.js

```diff
@@ -188,6 +187,10 @@ class GroupProfile extends Component {
     window.location.reload()
   }
 
+  _hideRegistrationModal () {
```

Comment:
I think having dedicated functions for setting a single state value like this might introduce a lot of clutter very quickly. keep in mind that I also have another PR coming up which adds another dialog to the groups page. Thoughts on a solution?

Reply:
I'm good with sticking to using `setState` directly instead of independent functions for it. I'll update that.

As for having multiple modals, it should be fine as long as they have independent display states, right?

Reply:
yeah I'm not worried about conflicting state values, I'm more worried about 30 lines of `(show/hide)Modal` functions for no good reason :P

tl;dr: I hate Java